MIGRATION_GUIDE.md

```diff
@@ -1,162 +1,21 @@
 This document describes breaking changes of `teloxide` crate, as well as the ways to update code.
 Note that the list of required changes is not fully exhaustive and it may lack something in rare cases.
 
-## 0.4 -> 0.5
```

Comment:
I don't think the old migration guide should be removed, there may still be some users with old bots, that were not migrated even to 0.5.

If `0.4 -> 0.5` will be below `0.5 -> 0.6` it won't hurt anyone and would be able to help somebody and show how `teloxide` changed over time.

This also makes more sense in the future, where `0.6 -> 0.7` migration guide would be able to mention `0.5 -> 0.6`: "The old dispatching system was removed, see `0.5 -> 0.6` for information about the new dispatching system blah blah blah".

Reply:
Fixed.